### ISSUE TITLE
WCAG AA (4.5:1)

### ISSUE THREAD

**Issue by Nompie1993** [Nompie1993/CleanCity#100]
**Test Steps:**

1.Run Axe DevTools on homepage.
2.Check "color-contrast" violations.

**Expected Result:**

1.All text passes WCAG AA (4.5:1).
2.Tool reports 0 contrast errors.

**Actual Result**

<img width="1342" height="583" alt="Image" src="https://github.com/user-attachments/assets/08135134-8276-4fd8-97ae-b3c0a0cecfeb" />
